scripts/xsslint/xsslint/linters.py

```diff
@@ -55,7 +56,7 @@ def _load_file(self, file_full_path):
         """
         with io.open(file_full_path, 'r') as input_file:
             file_contents = input_file.read()
-            return file_contents
+            return file_contents.decode(encoding='utf-8') if six.PY2 else file_contents
```

Comment:
`io.open()` will return unicode in both python2 and python3 this change is not needed.